app/src/main/java/fr/neamar/kiss/db/DBHelper.java

```diff
@@ -225,4 +228,55 @@ public static void removeShortcuts(Context context, String packageName) {
         db.close();        
     }
 
+    /**
+     * Insert new tags for given id
+     *
+     * @param context android context
+     * @param tag   tag to insert
+     * @param record  record to insert
+     */
+    public static void insertTagsForId(Context context, String tag, String record) {
+        SQLiteDatabase db = getDatabase(context);
+
+        ContentValues values = new ContentValues();
+        values.put("tag", tag);
+        values.put("record", record);
+        db.insert("tags", null, values);
+        db.close();
+    }
+
+
+    /* Delete
+     * Insert new item into history
+     *
+     * @param context android context
+     * @param tag   query to insert
+     * @param record  record to insert
+     */
+    public static void deleteTagsForId(Context context, String record) {
+        SQLiteDatabase db = getDatabase(context);
+
+        db.delete("tags", "record = '"+record+"'", null);
```

Comment:
You should use http://stackoverflow.com/a/30024262/2367848
Also spaces between `+`